src/engine/__tests__/is-authorized.test.js

```diff
@@ -0,0 +1,19 @@
+describe('is-authorized test', () => {
+  afterEach(() => {
+    jest.resetModules();
+  });
+
+  test('getAuthorization should return false', async () => {
```

Comment:
This is a good test, but we will probably need to pass a lot more arguments to simulate a real call to this function.